opencti-platform/opencti-graphql/src/schema/general.js

```diff
@@ -5,7 +5,6 @@ import { ENTITY_TYPE_CONTAINER_CASE } from '../modules/case/case-types';
 export const STIX_TYPE_RELATION = 'relationship';
 export const STIX_TYPE_SIGHTING = 'sighting';
 
-export const KNOWLEDGE_ORGANIZATION_RESTRICT = 'KNORGARESTRICT';
```

Comment:
?

Reply:
this constant was unused

Reply:
another 'KNOWLEDGE_ORGANIZATION_RESTRICT' (with different value) is defined in access.ts

Reply:
Oh yes, missed that 🥲